### ISSUE TITLE
Create secret with every parameter in parameter store under a certain prefix

### ISSUE THREAD

**Issue by ronlut** [external-secrets/kubernetes-external-secrets#566]
Does it make sense to add a feature to support importing all parameters under a certain prefix in param store / secrets manager?
This is kind of an enhancement to `dataFrom` feature.

What I would like to be able to achieve is the following:
### Parameter Store
* Parameter `/application/specific/prefix/a` with value `a`
* Parameter `/application/specific/prefix/b` with value `b`

... think 100 parameters

### ExternalSecret
```
apiVersion: 'kubernetes-client.io/v1'
kind: ExternalSecret
...
spec:
  backendType: systemManager
  dataFromPrefix:
    - /application/specific/prefix/
```

(This in functionality should be equal to:)
```
apiVersion: kubernetes-client.io/v1
kind: ExternalSecret
...
spec:
  data:
    - key: /application/specific/prefix/a
      name: a
    - key: /application/specific/prefix/b
      name: b
... 100 more
```

### Resulting Secret
```
apiVersion: v1
kind: Secret
type: Opaque
data:
  a: YQ== # a
  b: Yg== # b
```

What do you think?

**Comment by ectogon** [comment]
This would be excellent! It looks like there is related work in progress #238

**Comment by rjmsilveira** [comment]
For reference the feature was merged and was refactored in #603  :)

**Comment by Flydiverny** [comment]
As mentioned above this will be available in next release!
